### ISSUE TITLE
bom with identifiers

### ISSUE THREAD

**Issue by pbizzlr** [barkertrax/eurorack#1]
hey there ive had the Buffered Mult boards in my box for years and id love to finish it but the BOM doesnt seem to have the identifiers to match up with the board markings- can you advise?

**Comment by barkertrax** [comment]
Hi,
How’s it going?
I’m afraid it’s too long ago for me to remember the exact details :)
However, I looked out some photos that I posted on muffwiggler back in the day, which I’m attaching here. I think there should be enough information in them to work out which of the connectors go where, how the main board attaches, and the connections for the flying wires.
Yours
Mark

**Comment by pbizzlr** [comment]
hey thanks for getting back, no images attached tho? what thread are they in?

**Comment by barkertrax** [comment]
Ah. I just replied to the email github sent, so I guess images get stripped.
Using the website now, and attempting to re-attach
![8226_808bd51024_2](https://user-images.githubusercontent.com/2436774/98724029-0bc7c580-238b-11eb-9b09-03b7ddcc68ba.jpg)
![8226_808bd41024_1](https://user-images.githubusercontent.com/2436774/98724044-0e2a1f80-238b-11eb-9492-9db8f460434a.jpg)
![8226_808bd31024_1](https://user-images.githubusercontent.com/2436774/98724049-0ec2b600-238b-11eb-9926-697e948f38d6.jpg)
![8226_808bd21024_1](https://user-images.githubusercontent.com/2436774/98724050-0f5b4c80-238b-11eb-985e-ce7cd6938e83.jpg)
![8226_808bd11024_1](https://user-images.githubusercontent.com/2436774/98724053-0ff3e300-238b-11eb-8ea1-30e68c0b92db.jpg)

**Comment by pbizzlr** [comment]
thanks! but this is the BD module, its the buffered multiple i'm on about :)

**Comment by barkertrax** [comment]
Oops - that's embarrassing. Sorry!

I looked through my files for this module and I think these are the only useful things I have. The first is the original circuit I based my board on. The second is me making a parts list based on that circuit. I don't know that either are particularly useful to you, though!

Unfortunately, I don't have eagle installed anymore so I can't open the PCB layout. I'm guessing that's the key to this as the parts there will have their values, as well as part numbers.

![1060_buff_mult_1](https://user-images.githubusercontent.com/2436774/98729778-4afa1480-2393-11eb-84f9-df665d934868.jpg)

<img width="472" alt="image" src="https://user-images.githubusercontent.com/2436774/98729576-05d5e280-2393-11eb-90ec-ea659ab1a041.png">

**Comment by pbizzlr** [comment]
yes, this is exactly it. the bom with the designators on it . nice one my man they will see the light of day after all!!
